pkg/kudoctl/bundle/bundle.go

```diff
@@ -76,14 +76,113 @@ func (b tarBundle) GetCRDs() (*PackageCRDs, error) {
 }
 
 func (b fileBundle) GetCRDs() (*PackageCRDs, error) {
-	p, err := fromFolder(b.path)
+	p, err := FromFolder(b.path)
 	if err != nil {
 		return nil, errors.Wrap(err, "while reading package from the file system")
 	}
 	return p.getCRDs()
 }
 
-func fromFolder(packagePath string) (*PackageFiles, error) {
+// ToTarBundle takes a path to operator files and creates a tgz of those files with the destination and name provided
+func ToTarBundle(path string, destination string, name string, overwrite bool) error {
```

Comment:
would be nice to have some tests for this. You should probably use https://github.com/spf13/afero for that...